README.md

```diff
@@ -1,141 +1,481 @@
-[![Build Status](https://travis-ci.org/clickhouse/clickhouse-odbc.svg?branch=master)](https://travis-ci.org/clickhouse/clickhouse-odbc)
+# ODBC Driver for ClickHouse [![Build Status](https://travis-ci.org/clickhouse/clickhouse-odbc.svg?branch=master)](https://travis-ci.org/clickhouse/clickhouse-odbc) <!-- omit in toc -->
 
-If you are macos user see [MACOS.md](MACOS.md)
+## Introduction <!-- omit in toc -->
 
-## Cloning a Project with Submodules
+This is the official ODBC driver implementation for accessing ClickHouse as a data source.
 
-Please note - [git submodules](https://git-scm.com/book/en/v2/Git-Tools-Submodules) are used in this project. 
+For more information on ClickHouse go to [ClickHouse home page](https://clickhouse.yandex).
 
-When you clone such a project, by default you get the directories that contain submodules, but none of the files within them.
-So, in order to build the project, you need either:
-  * clone repo with all submodules altogether (use `--recursive`)
-```bash
-git clone --recursive https://github.com/clickhouse/clickhouse-odbc
+For more information on what ODBC is go to [ODBC Overview](https://docs.microsoft.com/en-us/sql/odbc/reference/odbc-overview).
+
+The canonical repo for this driver is located at [https://github.com/ClickHouse/clickhouse-odbc](https://github.com/ClickHouse/clickhouse-odbc).
+
+See [LICENSE](LICENSE) file for licensing information.
+
+## Table of content <!-- omit in toc -->
+
+- [Installation](#installation)
+- [Configuration](#configuration)
+  - [Troubleshooting: driver manager tracing and driver logging](#troubleshooting-driver-manager-tracing-and-driver-logging)
+- [Building from sources](#building-from-sources)
+- [Appendices](#appendices)
+  - [Run-time dependencies: Windows](#run-time-dependencies-windows)
```

Comment:
Does our installer install it? If so - no need to list.

Reply:
if user unpacks from a zip, then he has to make sure he have them.